verilog/tools/ls/vscode/README.md

```diff
@@ -26,5 +26,15 @@ The language server provides a couple of features from the [Verible SystemVerilo
    * **[&#x1F389; New]** `AUTOREG` – declares regs for outputs not connected to
      any module instance.
 
+## Prerequisite on your machine
+
+The Verible plug-in needs the `verible-verilog-ls` executable installed
+on your machine.
+
+Get a binary distribution for your Operating System at
+https://github.com/chipsalliance/verible/releases
+
+Or, since this is Open Source, you can also compile it yourself.
```

Comment:
How about:

"Alternatively, build Verible from source [link to instructions]"

Reply:
Ok, PTAL